### PR TITLE
[frinx-machine-operators,frinx-machine] Create frinx-machine-operators helm chart

### PR DESCRIPTION
**Pull request by JumasJM**
Moved operators to new chart to ensure that crds are installed before frinx-machine release.

- created new chart frinx-machine-operators
- moved required operators from frinx-machine chart
- operators chart needs to be installed before frinx-machine

Type: Improvement
JIRA: DEP-732

### CODE REVIEW

File: charts/frinx-machine/Chart.yaml [hunk 1]
```diff
@@ -3,22 +3,11 @@ name: frinx-machine
...
-
-  - name: kube-arangodb
-    version: 1.2.38
-    repository: https://arangodb.github.io/kube-arangodb
-    condition: kube-arangodb.enabled
-
-  - name: cloudnative-pg
-    repository: https://cloudnative-pg.github.io/charts
-    version: 0.20.2
-    condition: cloudnative-pg.enabled
```
Comment (Jozefiel) [inline charts/frinx-machine/Chart.yaml:11]: This is a breaking change, so chart version should be 8.0.0

File: charts/frinx-machine-operators/values.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      - --deployment.feature.ephemeral-volumes=true
+  rbac:
+    enabled: true
+    extensions:
+      monitoring: false
+      acs: false
+      at: false
+
+cloudnative-pg:
+  enabled: true
```
Comment (Jozefiel) [inline charts/frinx-machine-operators/values.yaml:16]: https://github.com/norwoodj/helm-docs can we use this tool to generate README.md?

File: charts/frinx-machine/README.md [hunk 3]
```diff
@@ -14,6 +14,14 @@ helm repo add frinx https://FRINXio.github.io/helm-charts
...
 ```
 
+## Install Operators Chart
+
+Before installing Frinx MACHINE, install required operators and crds
+
+```console
+helm install [RELEASE_NAME] frinx/frinx-machine-operators
+```
+
```
Comment (Jozefiel) [inline charts/frinx-machine/README.md:24]: Is OK to use the same release name for frinx machine and for operators?

File: .github/lint_test_config.yaml [hunk 4]
```diff
@@ -30,3 +30,5 @@ excluded-charts:
   - resource-manager
   # Temporary disabled for bug in Kafka
   - performance-monitor
+  # This chart has dependencies to CloudNativePg an Kube Arango
+  - frinx-machine-operators
```
Comment (Jozefiel) [inline .github/lint_test_config.yaml:34]: I think this should work by default

File: charts/frinx-machine/Chart.yaml (hunk 1 above)
Comment (JumasJM) [reply charts/frinx-machine/Chart.yaml:11]: Bumped.

File: charts/frinx-machine-operators/values.yaml (hunk 2 above)
Comment (JumasJM) [reply charts/frinx-machine-operators/values.yaml]: Helm-docs used with manual addition.

File: .github/lint_test_config.yaml (hunk 4 above)
Comment (JumasJM) [reply .github/lint_test_config.yaml]: Removed.

File: charts/frinx-machine/README.md (hunk 3 above)
Comment (JumasJM) [reply charts/frinx-machine/README.md]: Added -operators after [RELEASE_NAME].

**Pull request merged**